theme_notes/views/snippets/s_company_team_spotlight.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_spotlight" inherit_id="website.s_company_team_spotlight" name="Notes s_company_team_spotlight">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="o_cc5 o_colored_level" remove="o_cc1" separator=" "/>
```

Comment:
```suggestion
        <attribute name="class" add="o_cc5" remove="o_cc3" separator=" "/>
```
or 
```suggestion
        <attribute name="class" add="o_cc5" remove="o_cc4" separator=" "/>
```
or `o_cc1` with a background shape

No need for `o_colored_level` it should be on the section of the default snippet